websites/Z/ZeroTwo/presence.ts

```diff
@@ -0,0 +1,19 @@
+const presence = new Presence({
+    clientId: "752067809214857298"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+  
+  presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+      largeImageKey: "logo"
+    };
+  
+    if ((await presence.getSetting("incognito")) === false) {
+      presenceData.details = document.getElementById("premid").innerText;
```

Comment:
Some pages on the website don't have this element, so you should probably support those too (e.g. premium)

Reply:
Why was this marked as resolved?

Reply:
Then this should only be resolved when that’s done